### ISSUE TITLE
Would you please provide a training log of RADD-small model?

### ISSUE THREAD

**Issue by dingfangyu** [ML-GSAI/RADD#5]
Dear authors, 

Thanks for your great work.
We are trying to follow this work, so it will be great for us to access the original training log (e.g. of RADD-small model for 400k steps), would you like to provide it? Thank you!

Best regards

**Comment by dingfangyu** [comment]
Regarding the training setting, we are curious about the config of optimizer, is this version of config adopted in the original training process?

optim:
  weight_decay: 0
  optimizer: AdamW
  lr: 3e-4
  beta1: 0.9
  beta2: 0.999
  eps: 1e-8
  warmup: 2500
  grad_clip: 1.

where the weight decay is 0, different from the reported version of 0.03 in RADD paper

**Comment by JingyangOu** [comment]
Thank you for your interest in our work!

[Training Log](https://drive.google.com/file/d/1Wo5ZFS_2aDbex1-mA_KhQTrNxXv5eH8i/view?usp=sharing) : This is the log of RADD-small model for 1000k steps, including the results for first 400k steps. 

Regarding the optimizer config — yes, your understanding is correct.

**Comment by dingfangyu** [comment]
Thanks for your response!

**Comment by dingfangyu** [comment]
Hi @JingyangOu, do you use a naive random timestep sampler in the training process of all the models reported in the RADD paper? I believe that the naive sampler is beneficial to obtain a lower loss compared with vdm or decoupled in https://github.com/ML-GSAI/RADD/blob/33fce537c0061118d72fa1d0f3de1cef90bf1d56/losses.py#L7 , although the laters are claimed stabler in training in other works.

**Comment by JingyangOu** [comment]
Yes, all the models reported in the RADD paper are trained with the naive random timestep sampler.
